localization/english/replace/modifiers_l_english.yml

```diff
@@ -0,0 +1,2134 @@
+﻿l_english:
```

Comment:
When replacing a string. Don't copy over the whole file. Only add in this replace files the string you changed.
Then it will use the original for the rest.